Vagrantfile

```diff
@@ -330,6 +418,23 @@ Vagrant.configure(2) do |config|
     b.vm.provision "run tests", :type => :shell, :privileged => false, :inline => run_tests("buster64", ".*none.*")
   end
 
+  config.vm.define "stretch64" do |b|
+    b.vm.box = "debian/stretch64"
+    b.vm.provider :virtualbox do |v|
+      v.memory = 1024 + $wmem
+    end
+    b.vm.provision "fs init", :type => :shell, :inline => fs_init("vagrant")
+    b.vm.provision "packages debianoid", :type => :shell, :inline => packages_debianoid("vagrant")
+    b.vm.provision "install source dependencies", :type => :shell, :privileged => true, :inline => install_source_dependencies("vagrant")
+    b.vm.provision "install pyenv", :type => :shell, :privileged => false, :inline => install_pyenv("stretch64")
+    b.vm.provision "install pythons", :type => :shell, :privileged => false, :inline => install_pythons("stretch64")
+    b.vm.provision "build env", :type => :shell, :privileged => false, :inline => build_pyenv_venv("stretch64")
```

Comment:
hmm, if we first install_pythons, we'll have a recent python even on stretch, which could be used for install_source_dependencies.

Reply:
I thought so too and had it working that way before but it makes it a bit awkward with pyfuse3 and openssl. Two functions are needed in this case.

A recent openssl is required to build python and must be installed before pyenv runs.

Pyfuse3 needs root to setuid for fusermount3. The pyenv runs with user vagrant but it does work with sudo ninja install.

Reply:
ah, ok, so guess we keep it as is.